imports/plugins/core/ui-tagnav/client/components/tagNav.js

```diff
@@ -53,6 +68,48 @@ class TagNav extends Component {
     };
   }
 
+  /**
+  * renderShopSelect
+  * @method
+  * @summary Handles shop options display on mobile view
+  * @return {JSX} component containing dropdown
+  */
+  renderShopSelect() {
```

Comment:
It might be better now, to break this chunk into a component, since it's being used in both `dashboard/client/components/toolbar.js` and `ui-tagnav/client/components/tagNav.js`.

Reply:
@efalayi I think this is the last change to make and we can merge this in